iam-policy-autopilot-policy-generation/src/api/model.rs

```diff
@@ -49,6 +51,20 @@ pub struct AwsContext {
     pub account: String,
 }
 
+/// Exposes git version and commit hash for boto3 and botocore
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct GitSubmoduleMetadata {
+    /// the git commit hash
+    #[serde(rename = "gitCommit")]
```

Comment:
We use `PascalCase` for JSON throughout the project, let's use `#[serde(rename_all="PascalCase")]` on the struct and then specialize for `GitCommit`